main.js

```diff
@@ -0,0 +1,163 @@
+var photos = [];
+
+
+
+$(function() {
+  $("#search-form").on('submit', search);
+});
+
+function search(event) {
+  event.preventDefault();
+  clearSearchResults();
+  var input = $("#query");
+  var userQuery = input.val() || input.attr("placeholder");
+  console.log("searching for:", userQuery);
+  $.get('https://www.reddit.com/search.json', {
+    q: userQuery
+  }).done(function(response) {
+    console.log(response);
+
+    var results = response.data.children;
+    for (var i = 0; i < results.length; i++) {
+      var result = results[i].data.url;
+      photos.push(result);
+      console.log(photos);
+    }
+    setInterval(function(){
+      var index = 0;
```

Comment:
I think the reason the slideshow isn't working here is that the index is scoped to the function. Every time the function fires, it creates a new variable named index, sets it to 0, and then displays photos[index] (first photo every time.)
If you set index outside of the function, this should work, except then you'll have to handle when index > the number of photos (Hint: modulus)